tests/utility/hash_test.cu

```diff
@@ -48,6 +48,77 @@ __host__ __device__ bool check_hash_result(typename Hash::argument_type const& k
   return (h(key) == expected);
 }
 
+// Overload for hash functions without a seed
+template <typename Hash>
+__host__ __device__ bool check_hash_result(typename Hash::argument_type const& key,
+                                           typename Hash::result_type expected) noexcept
+{
+  Hash h;
+  return (h(key) == expected);
+}
+
```

Comment:
I like this solution, and the idea of doing it in a different PR since it affects so many non-identity_hash tests. Perhaps it makes sense to knock this and #518 out in the same PR

Reply:
Opened issue #521 to cover this